README.md

```diff
@@ -179,6 +179,15 @@ extension UIImage {
 
 For SwiftUI a Swift file will be created to use images from the code.
 
+#### Images with multiplue Idiom
+when you want to specify image for different platform usage (iPhone, iPad, Mac...etc), you should add extra `~` mark (add the platform type) to the component your figma file, for example add `~ipad` to the image:
```

Comment:
1. In this section we show result of an export command. So, I think we should leave only this text and one image here:

```
If name of an image contains idiom at the end (e.g. ~ipad), it will be exported like this:

<img src="images/ios_image_idiom_xcode.png"/>
```

2. Add the following text to the «Design requirements»/«For figma-export images» section.

```
If you want to specify image variants for different devices (iPhone, iPad, Mac etc.), add an extra `~` mark with idiom name. For example add `~ipad` postfix:

<img src="images/ios_image_idiom_figma.png"/>

```